protocol/consistencychecks_test.go

```diff
@@ -64,31 +66,19 @@ func TestVerifyRegistrationResponseWithTB(t *testing.T) {
 
 	cc := NewCC(d.LatestSTR(), true, pk)
 
-	if doRequestAndVerify(d, cc, RegistrationType, "alice") != CheckPassed {
-		t.Fatal("Unexpected verification result")
+	if e1, e2 := registerAndVerify(d, cc, alice, key); e1 != ReqSuccess || e2 != CheckPassed {
+		t.Error(e1)
+		t.Error(e2)
 	}
 
 	if len(cc.TBs) != 1 {
 		t.Fatal("Expect the directory to return a signed promise")
 	}
 
 	// test error name existed
-	// FIXME: Check that we got an ReqNameExisted
-	if doRequestAndVerify(d, cc, RegistrationType, "alice") != CheckPassed {
-		t.Fatal("Unexpected verification result")
-	}
-
-	// test error name existed with different key
```

Comment:
Did you mean to drop this test?  It would return `ReqNameExisted` and some sort of `CheckBad*`, which we probably want to improve to `CheckBadBinding` or so.  We don't actually have an `ErrBindingsDiffer` yet, do we?

We can also leave a pointer to #133